config/software/libtool.rb

```diff
@@ -32,6 +33,11 @@
     patch source: "v2.4.ppc64le-configure.patch", plevel: 1
   end
 
+  # Update config.guess to support newer platforms (like aarch64)
+  if version == "2.4"
+    patch source: "config.guess_2015-09-14.patch", plevel: 0
+  end
```

Comment:
This patch conflicts with ``v2.4.ppc64le-configure.patch", plevel: 1`` above. Removing that patch seems to fix it.